docs/uma-tokenholders/voter-dApp.md

```diff
@@ -24,19 +24,17 @@ You must commit `AND` reveal your vote in order for it to be counted
 
 3. Commit period: You’ll see active votes in the “Active Requests” table. 
 
-![Activerequests](/docs/developers/votingdApp_activerequests.png)
-- Type your vote into each row by using the “edit” button
-- Check the box in each row 
-- Click the “save” button
+![Activerequests](/docs/voterdappguide/edit-votes.png)
+![Confirmationbar](/docs/voterdappguide/sign-commit.png)
+- Type your vote for each vote under the **"current vote"** dropdown.
```

Comment:
This formatting is a little funky, where the bullets are at the same level as the numbered steps. You can remove the space between the bullets and numbers and this should make them sub-bullets.

Reply:
Updated!